contract_invoice_merge_by_partner/README.rst

```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=================================
+Contract Invoice Merge By Partner
+=================================
+
+This module merges same partner invoices generated by contracts.
+
+Installation
+============
+
+To install this module you need *account_invoice_merge*, available in:
+
+* Install repository `OCA/account-invoicing <https://github.com/OCA/account-invoicing>`_.
+
+Usage
+=====
+
```

Comment:
Go to Sales > Customer > Choose a partner and in the tab "Sales & Purchases" activate check " Merge contracts invoices "